sufia-models/app/jobs/create_derivatives_job.rb

```diff
@@ -8,7 +8,11 @@ def run
     if generic_file.video?
       return unless Sufia.config.enable_ffmpeg
     end
-    generic_file.create_derivatives
+    status = Timeout::timeout(Sufia.config.derivatives_timeout) do
```

Comment:
HydraDerivatives sets a timeout for each item that it creates, unfortunately image processing hung and that does not have a timeout option.  Additionally I just want to be certain the entire derivatives generation does not go on forever. The timeouts are way down in the weeds in hydra derivatives, so other portions could hang and those timeouts may not catch the thing hanging.